### ISSUE TITLE
`grunt test` doesn't work

### ISSUE THREAD

**Issue by SCdF** [medic/cht-core#6317]
The `grunt test` target fails, because:
 - It tries to switch a unit testing environment variable on for unit tests, then off for integration tests
 - However, files that read that environment variable only do so on initial load, and so their decision is cached between unit and int tests

You can work around it by running each inner target individually in separate executions, and so separate node vms.

**Comment by garethbowen** [comment]
Is this a useful task? I _always_ run the individual tasks because running the whole suite would take too long. If nobody uses it I think we just drop it...

Alternatively I'd like to get rid of the environment variable altogether and detect testing by checking some global variable instead.

**Comment by SCdF** [comment]
Yeah clearly no one uses it otherwise this would have been raised before. I like to run everything when I'm refactoring something especially if the tests for it aren't great, while I go make a coffee or whatever.

I've chucked that into npm as a run action. We can call it whatever feels appropriate, but I've used test. Or if we don't want to do that I'll just ctrl+r that out of my shell history whenever I need it ;-)

https://github.com/medic/cht-core/pull/6319
